audit/projects/k8s-infra-e2e-boskos-001/buckets/kubernetes-staging-485128143e-asia/cors.txt

```diff
@@ -0,0 +1 @@
+gs://kubernetes-staging-485128143e-asia/ has no CORS configuration.
```

Comment:
I will admit, I am _not_ used to seeing `-asia` and `-eu` buckets get created by kube-up.  It supports it, but I'm not used to seeing it.  Has something changed?

https://github.com/kubernetes/kubernetes/blob/703e2bd2ebae1cc7737c12344a9e6fb13a6019a9/cluster/gce/util.sh#L336

https://github.com/kubernetes/kubernetes/blob/703e2bd2ebae1cc7737c12344a9e6fb13a6019a9/cluster/gce/util.sh#L255-L281

Looks like `RELEASE_REGION_FALLBACK` needs to be `true` for this to happen
- it defaults to false in kube-up...
- nothing currently sets it to true... https://cs.k8s.io/?q=RELEASE_REGION_FALLBACK&i=nope&files=&excludeFiles=&repos=

Reply:
nit: I think we should consider dumping bucket metadata as part of audit
```
$ gsutil ls -L -b gs://kubernetes-staging-485128143e-asia
gs://kubernetes-staging-485128143e-asia/ :
	Storage class:			STANDARD
	Location type:			multi-region
	Location constraint:		ASIA
	Versioning enabled:		None
	Logging configuration:		None
	Website configuration:		None
	CORS configuration: 		None
	Lifecycle configuration:	None
	Requester Pays enabled:		None
	Labels:				None
	Default KMS key:		None
	Time created:			Sun, 09 Aug 2020 17:57:04 GMT
	Time updated:			Sun, 09 Aug 2020 17:57:04 GMT
	Metageneration:			1
	Bucket Policy Only enabled:	False
# ... snip ...
```

So the "time created" tells me whatever this was, it happened long ago.  Well past our log retention on prow.k8s.io, gs://kubernetes-jenkins, or k8s-infra-prow-build.

Maybe consider deleting these and revisiting if they start popping up again

Reply:
No wait, I lied... still definitely in use.  It has to do with where the cluster is created.
```
spiffxp@spiffxp-macbookpro:test-infra (master)$ ag '(asia|europe)-'
config/jobs/kubernetes/sig-network/sig-network-misc.yaml
335:      - --gcp-zone=asia-southeast1-a
372:      - --gcp-zone=asia-southeast1-a

config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml
158:      - --gcp-zone=asia-southeast1-a
189:      - --gcp-zone=asia-southeast1-a

config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml
849:      - --gcp-zone=europe-west1-c

config/jobs/kubernetes/sig-node/containerd.yaml
600:      - --gcp-zone=asia-southeast1-a
```

OK! carry on then